terraform/modules/quorum-cluster-region/quorum-cluster-region-backup-event.tf

```diff
@@ -0,0 +1,399 @@
+    handler          = "BackupLambda" # Name of Go package after unzipping the filename above
+    role             = "${aws_iam_role.iam_for_BackupLambda.arn}"
+    runtime          = "go1.x"
+    source_code_hash = "${sha256("file(${var.aws_region}-${var.BackupLambda_output_path})")}" # 
+    timeout          = 300
+
+    vpc_config {
+       subnet_ids = ["${aws_subnet.BackupLambdaAccessInternet.id}"]
+       security_group_ids = ["${aws_security_group.allow_all.*.id}"]
+    }
+
+    environment {
+        variables = {
+            NetworkId = "${var.network_id}"
+            Bucket = "${aws_s3_bucket.quorum_backup.id}"
+            Key = "${var.enc_ssh_key}"
+            SSHUser = "ubuntu"
+        }
+    }
+}
+
+// "
+
+resource "aws_iam_role" "iam_for_BackupLambda" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  name = "iam_for_BackupLambda-${var.network_id}-${var.aws_region}"
+# See also https://aws.amazon.com/blogs/compute/easy-authorization-of-aws-lambda-functions/
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "lambda.amazonaws.com",
+        "Service": "events.amazonaws.com",
+        "Service": "sns.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_policy" "BackupLamba_Permissions" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  path = "/"
+  description = "IAM policy for accesing EC2 and S3 buckets from Lambda"
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "ec2:*"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    },
+    {
+      "Action": ["s3:*"],
+      "Effect": "Allow",
+      "Resource": [
+        "${aws_s3_bucket.quorum_backup.arn}",
+        "${aws_s3_bucket.quorum_backup.arn}/*"
+      ]
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy_attachment" "BackupLambda1" {
+   count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+   role = "${aws_iam_role.iam_for_BackupLambda.name}"
+   policy_arn = "${aws_iam_policy.BackupLamba_Permissions.arn}"
+}
+
+
+resource "aws_iam_policy" "BackupLambda_logging" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  name = "BackupLambda-${var.network_id}-${var.aws_region}"
+  path = "/"
+  description = "IAM policy for logging from a lambda"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "logs:*"
+      ],
+      "Resource": "arn:aws:logs:*:*:*",
+      "Effect": "Allow"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy_attachment" "BackupLambda2" {
+   count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+   role = "${aws_iam_role.iam_for_BackupLambda.name}"
+   policy_arn = "${aws_iam_policy.BackupLambda_logging.arn}"
+}
+
+# Zip the Backup Lambda
```

Comment:
Per my earlier email I think we should be downloading already-zipped releases from GitHub

Reply:
Now retrieving zip from GitHub